contracts/stabletoken_ERC20.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
```

Comment:
Done 👍

Reply:
should it mention the erc20 toekn as well?
Also let's rename the erc20 to: "ERC20.sol" /"ERC20_ABI.sol"

Reply:
Will we have different erc20 implementations ? Should it be left as stableToken so it's not misinterpreted ?

Reply:
I guess what would make the most sense, but not sure if it's feasible, is to move the bytecode and the sol file to the integration folder, since these are only used for testing.
What we need in the main folder is just how to interpret ERC20 transactions